guillotina/db/storages/pg.py

```diff
@@ -782,21 +782,21 @@ def objects_table_name(self):
 
     @restart_conn_on_exception
     async def open(self):
-        conn = await self.pool.acquire(timeout=self._conn_acquire_timeout)
+        conn = await self.pool.acquire()
         return conn
 
     async def close(self, con):
         try:
-            await shield(asyncio.wait_for(self.pool.release(con, timeout=1), 1))
+            await shield(self.pool.release(con, timeout=1))
         except (
             asyncio.CancelledError,
-            RuntimeError,
             asyncio.TimeoutError,
```

Comment:
asyncpg 0.19 has rewritten the RuntimeError as InterfInterfaceError

Reply:
lovely

Reply:
can you leave in `RuntimeError` in case we aren't on 0.19?